scopesim/tests/mocks/basic_instrument/default.yaml

```diff
@@ -1,8 +1,9 @@
 # Default config file for setting up the instrument
-object : observation
+object: configuration
 alias : OBS
```

Comment:
I suggest to also change the alias (but that might break things?)

Reply:
This top section in `default.yaml` (aka the defyam as it's called in some places) seems to always have `alias: OBS`, see e.g. MICADO and METIS in the IRDB...

Reply:
at least it is consistent now